distributed/scheduler.py

```diff
@@ -5393,7 +5405,7 @@ async def gather(self, comm=None, keys=None, serializers=None):
     def clear_task_state(self):
         # XXX what about nested state such as ClientState.wants_what
         # (see also fire-and-forget...)
-        logger.info("Clear task state")
+        logger.debug("Clear task state")
```

Comment:
I changed this log level since it felt kind of wrong. This prints a `Clear task state` to the console whenever I call `dask-scheduler`. That's currently failing the tests. I assume this is OK?